dashboard/test/ui/features/header.feature

```diff
@@ -0,0 +1,65 @@
+@no_mobile
+Feature: Header navigation bar
+
+Scenario: Signed out user in English should see 6 header links
+  Given I am on "http://code.org/"
+  And I dismiss the language selector
+  And I wait to see "#headerlinks"
+  And I wait to see ".headerlink"
```

Comment:
Once you've waited for one indicator that everything has loaded, I think it's more readable to just race through everything that you know will be there.  I'd just use waits when it's actually necessary.